### ISSUE TITLE
Value Stream Map is not updating pipeline status

### ISSUE THREAD

**Issue by JeroenOortwijn** [gocd/gocd#13274]
##### Issue Type

- Bug Report
- Feature proposal

##### Summary

When you look at a Value Stream Map (VSM) while it is running the pipelines, it doesn't update the pipeline statuses.

###### Basic environment details

* Go Version: `24.4.0 (19686-4e34832acbaf77d46bca61ccc4b0f8d458831a31)`
* JAVA Version: `21.0.5`
* OS: `Linux 5.4.0-200-generic`
* Browser vendor and version (if relevant): `Opera One (version: 114.0.5282.185) - Chromium version: 128.0.6613.186`

##### Steps to Reproduce

1. Trigger a pipeline that will then trigger other pipelines.
2. Open the VSM of the pipeline while the other pipelines are running.
3. Notice that the running state of the pipelines don't get updated.
4. Notice that the running state of the pipelines on the dashboard do get updated.

##### Expected Results
I would like the running state of the pipelines in the VSM to also get updated.

##### Actual Results
The running state of the pipelines in the VSM don't get updated.

##### Code snippets/Screenshots
Screenshot of VSM still showing old state:
![image](https://github.com/user-attachments/assets/0c272177-c08a-4711-b195-1d3812e0fdd9)

Screenshot of dashboard showing current state:
![image](https://github.com/user-attachments/assets/0497158c-064b-421c-a07e-d542ab494182)

**Comment by chadlwilson** [comment]
I don't believe the VSM was ever a live view. Do you have evidence otherwise?

**Comment by JeroenOortwijn** [comment]
No, this was the first time I looked at this. 😄
That's why I didn't know which one to chose, bug report or feature proposal.

**Comment by chadlwilson** [comment]
Ahh okay, fair enough. There is an old ticket for this but it was closed, possibly due to some clean up.

**Comment by JeroenOortwijn** [comment]
Ah, my bad, I forgot to search the closed tickets for a similar issue.

**Comment by chadlwilson** [comment]
I'm not sure re-opening it is of much help anyway: https://github.com/gocd/gocd/issues/449 :-)
